ckan/templates/user/edit_user_form.html

```diff
@@ -8,7 +8,11 @@
     {% block core_fields %}
       <fieldset>
         <legend>{{ _('Change details') }}</legend>
-        {{ form.input('name', label=_('Username'), id='field-username', value=data.name, error=errors.name, classes=['control-medium'], attrs={'readonly': '', 'class': 'form-control'}) }}
+        {% if g.userobj.sysadmin %}
```

Comment:
should put this behind a feature flag to push it upstream.

Reply:
We could, and if that's what it takes to be accepted upstream then I would, but it's restricted to sysadmins in any case, so I'm not sure there's really a need for anyone to switch it off.

Reply:
Discussion was at https://github.com/ckan/ckan/issues/4193

Reply:
Created https://github.com/ckan/ckan/pull/8122